src/main/java/com/karankumar/bookproject/ui/book/FormErrors.java

```diff
@@ -0,0 +1,12 @@
+package com.karankumar.bookproject.ui.book;
+
+public class FormErrors {
```

Comment:
Since these errors are only for the `BookForm`, could you refactor the name to `BookFormTests` (see the comment I made about why I don't think having one class for form errors is the best idea here)